metabase/metabase
# Add custom font settings

Epic https://github.com/metabase/metabase/issues/22582

Adds a custom font settings widget. Note that:
- The page is no longer reloaded when the font is changed. Changes are applied automatically.
- Signed embedding should be adjusted in a follow-up PR to work correctly

How to test:
- Run Metabase EE
- Go to `/admin/settings/whitelabel`
- Change the font to `Custom`
- Set `Regular 400` to `https://fonts.gstatic.com/s/lora/v24/0QIvMX1D_JOuMwr7I_FMl_E.woff2`
- Make sure the font is applied
- Change the font to `Lora`
- Make sure there is no visual difference since the file above is for `Lora`
- Change the font back to `Lato`
- Make sure the change is applied

<img width="1375" alt="Screenshot 2022-07-04 at 14 46 23" src="https://user-images.githubusercontent.com/8542534/177148866-c8a5da7a-bfef-4b68-b460-1cd4c6613fd6.png">
